### PR TITLE
Added Token Authentication

### PR DESCRIPTION
**Pull request by mephenor**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -35,7 +35,7 @@ zip_safe = False
 include_package_data = True
 packages = find:
 install_requires =
-    ghga-service-commons[api]==0.3.2
+    ghga-service-commons[api]==0.4.1
```
Comment (TheByronHimes) [inline setup.cfg]:
```suggestion
    ghga-service-commons[api]==0.4.2
```

File: setup.cfg [hunk 2]
```diff
@@ -35,7 +35,7 @@ zip_safe = False
...
     ghga-event-schemas==0.11.0
```
Comment (TheByronHimes) [inline setup.cfg]:
```suggestion
    ghga-event-schemas==0.13.2
```

**Review by TheByronHimes — DISMISSED** [review]
Dependency versions need to be updated, looks good otherwise

**Review by TheByronHimes — COMMENTED** [review]
Just an idea: would be nice to have a test verifying that invalid auth does **not** allow the request to proceed.

**Review by TheByronHimes — DISMISSED** [review]

**Comment by mephenor** [comment]
Good idea, added in c12bbb0

**Review by TheByronHimes — APPROVED** [review]

**Pull request merged**